docs/quickstart.rst

```diff
@@ -0,0 +1,553 @@
+ 
+.. code-block:: python
+
+	def process_local_login(self, data):
+	    username = data['username']
+	    password = self.settings['identities'].get(username)
+	    if password is None:
+		raise AuthenticationFailed('No such user')
+	    if password != data['password']:
+	        raise AuthenticationFailed('Invalid password.')
+	    auth_info = AuthInfo(self, username=data['username'])
+	    return self.multipass.handle_auth_success(auth_info)
+
+
+Identification
+~~~~~~~~~~~~~~~~
+
+The next step after successful authentication is assigning an identity to the user. That's a job of identity provider so let's have a look how to implement one.
+
+Your identity provider should inherit from ``IdentityProvider`` base class. The most important method it has to implement is ``get_identity_from_auth``. Which accepts ``AuthInfo`` object as an argument and returns the corresponding identity (object of ``IdentityInfo``) based on identifier. 
+
+In our example we search the ``'identities'`` dictionary that we specified in configuration and look for the identity with matching identifier (``'username'`` in our case).
+
+
+.. code-block:: python
+
+	class StaticIdentityProvider(IdentityProvider):
+
+	    def get_identity_from_auth(self, auth_info):
+		identifier = auth_info.data['username']	
+		user = self.settings['identities'].get(identifier)
+		if user is None:
+		    return None
+		return IdentityInfo(self, identifier, **user)
+			
+Other methods that should be implemented to ensure the full Multipass functionality can be found further in this guide. See :ref:`identities` and :ref:`groups`
+
+Now let's get back to the identification process. 
+Once ``handle_auth_success`` is called, it collects the identities linked to the user using the ``get_identity_from_auth`` we just mentioned. 
+Once identities are successfully collected, the method registered via ``'@multipass.identity_handler'`` decorator is called. A method with this decorator must be implemented in your application. Check the :ref:`identity_handler` part of this documentation for more info.
+
+Failed authentication
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In case the authentication was unsuccessful, and ``MultipassException`` was raised, ``handle_auth_error`` flashes the ``MULTIPASS_FAILURE_MESSAGE`` and if ``redirect_to_login`` argument is set, it redirects to ``MULTIPASS_LOGIN_ENDPOINT``
+
+
+Local provider logout
+~~~~~~~~~~~~~~~~
+
+Process of logging out local providers is the same as with external providers.
+Please check the :ref:`external_logout` part of this guide.
+
+.. _identities:
+
+Identities
+----------
+
+To retrieve ``IdentityInfo`` object  your  ``IdentityProvider`` must implement ``get_identity`` method. Example from ``StaticIdentityProvider``:
```

Comment:
"To retrieve **an** ``IdentityInfo`` object *comma*  your  ``IdentityProvider`` must implement **the** ``get_identity`` method. Example from ``StaticIdentityProvider``"